parse.py

```diff
@@ -46,8 +45,10 @@
 
     instr_dict = create_inst_dict(extensions, include_pseudo)
 
-    with open("instr_dict.yaml", "w") as outfile:
-        yaml.dump(add_segmented_vls_insn(instr_dict), outfile, default_flow_style=False)
+    with open("instr_dict.json", "w") as outfile:
+        json.dump(
+            add_segmented_vls_insn(instr_dict), outfile, ensure_ascii=False, indent=2
+        )
```

Comment:
@Timmmm Also, I could see this: any specific reason to mention `ensure_ascii` as `False` and why we need to explicitly state that parameter.

Reply:
It means it will use UTF-8 encoding for the JSON, however I don't think we'll have any non-ASCII anyway so I'll remove it for simplicity.